testament/important_packages.nim

```diff
@@ -31,27 +32,34 @@ pkg1 "cligen", "nim c --path:. -r cligen.nim"
 pkg1 "combparser", "nimble test --gc:orc"
 pkg1 "compactdict"
 pkg1 "comprehension", "nimble test", "https://github.com/alehander42/comprehension"
-# pkg1 "criterion" # pending https://github.com/disruptek/criterion/issues/3 (wrongly closed)
+# pending https://github.com/nim-lang/Nim/issues/16447
+#pkg1 "criterion", "nimble test", "https://github.com/disruptek/criterion"
 pkg1 "dashing", "nim c tests/functional.nim"
 pkg1 "delaunay"
 pkg1 "docopt"
 pkg1 "easygl", "nim c -o:egl -r src/easygl.nim", "https://github.com/jackmott/easygl"
 pkg1 "elvis"
 pkg1 "fidget"
 pkg1 "fragments", "nim c -r fragments/dsl.nim"
+# pending https://github.com/nim-lang/Nim/issues/16447
+#pkg1 "frosty", "nimble test", "https://github.com/disruptek/frosty"
 pkg1 "gara"
 pkg1 "glob"
 pkg1 "ggplotnim", "nim c -d:noCairo -r tests/tests.nim"
-# pkg1 "gittyup", "nimble test", "https://github.com/disruptek/gittyup"
+# pending https://github.com/nim-lang/Nim/issues/16447
+#pkg1 "gittyup", "nimble test", "https://github.com/disruptek/gittyup"
 pkg1 "gnuplot", "nim c gnuplot.nim"
-# pkg1 "gram", "nim c -r --gc:arc --define:danger tests/test.nim", "https://github.com/disruptek/gram"
-  # pending https://github.com/nim-lang/Nim/issues/16509
```

Comment:
* keep this `pending https://github.com/nim-lang/Nim/issues/16509` ?
* space after `#` (in all your edits)